README(site plan).md

```diff
@@ -0,0 +1,10 @@
+Overview (b8-8.5 audis)
+    The practical useage of this website would be for one place for all of audi parts aftermaket and OEM for the cars. it would be a static site that will show the lowest price that a peticular brand is offering 
+Goals
+    The main goal of this website will have all after market and OEM parts for cars on one page
+Target users 
+    the users will be the owners who  want to work on there cars at home to fix them or to make them faster or better 
+content
+    the site will have three pages, page one will be the home page ingormation about the b8-8.5 audi's. page two will have external body mods that could be donr to the cars. page three will have the mechanical modification or OEM replacements that a preson can fix at home.
```

Comment:
Site needs a minimum of 4 pages. I would replace the parts/prices page with something more realistic for this project.